drivers/vulkan/vulkan_context.h

```diff
@@ -328,6 +326,9 @@ class VulkanContext {
 
 	VulkanContext();
 	virtual ~VulkanContext();
+
+	VulkanContext(const VulkanContext &) = delete;
+	VulkanContext &operator=(const VulkanContext &) = delete;
```

Comment:
Deleting default members may be fine, but I don't think that matches current Godot coding style. I mean, many classes and structs may be copyable even when that may make no sense and we don't care. If we really want to enforce it, that should be done across the whole codebase.